layers/+spacemacs/spacemacs-org/local/space-doc/space-doc.el

```diff
@@ -102,11 +106,46 @@ It is set by `spacemacs//space-doc-set-space-doc-cache'.")
 
 (defun spacemacs//space-doc-hide-line-numbers (&optional enable)
   "If ENABLE is non-nil then toggle off the line numbers."
+  ;; FIXME: Can we prevent it from producing a message?
   (if enable
       (spacemacs/toggle-line-numbers-off)
     (when dotspacemacs-line-numbers
       (spacemacs/toggle-line-numbers-on))))
 
+(defun spacemacs//space-doc-modf-center-buffer (&optional enable)
+  "If ENABLE is non-nil then toggle on buffer centering."
+  (if enable (let ((fringe-width (-(/ (- (frame-pixel-width)
+                                         (car(window-text-pixel-size)))
+                                      ;; 50 * 2 extra pixels for the font shenanigans.
+                                      2) 50)))
+               (delete-other-windows)
+               (set (make-local-variable 'spacemacs--space-doc-origin-left-fringe-width)
+                    (or left-fringe-width 0))
+               (set (make-local-variable 'spacemacs--space-doc-origin-right-fringe-width)
+                    (or right-fringe-width 0))
+               (when (> fringe-width 0)
+                 (setq left-fringe-width fringe-width
+                       right-fringe-width fringe-width)
+                 (setq-local split-width-threshold nil)
```

Comment:
@syl20bnr  hmm.  Apparently nil doesn't work. But `(setq-local split-width-threshold 2000)` will do the trick.